pilot/pkg/config/kube/ingress/conversion.go

```diff
@@ -249,8 +259,12 @@ func ingressBackendToHTTPRoute(backend *v1beta1.IngressBackend, namespace string
 	if backend.ServicePort.Type == intstr.Int {
 		port.Number = uint32(backend.ServicePort.IntVal)
 	} else {
-		// Port names are not allowed in destination rules.
-		return nil
+		resolvedPort, err := resolveNamedPort(backend, namespace, client)
+		if err != nil {
+			log.Infof("failed to resolve named port %s, error: %s", backend.ServicePort.StrVal, err.Error())
```

Comment:
nit: You can also use `%v` to avoid calling `err.Error()`

```go
log.Infof("failed to resolve named port %s, error: %v", backend.ServicePort.StrVal, err)
```

Reply:
Done :-)